app/src/main/AndroidManifest.xml

```diff
@@ -3,7 +3,11 @@
     package="de.robv.android.xposed.installer"
     xmlns:android="http://schemas.android.com/apk/res/android">
 
-    <uses-permission android:name="android.permission.ACCESS_SUPERUSER"/>
+    <uses-sdk
```

Comment:
Why are you declaring uses-sdk here when is already declared in build.gradle?
And why min-sdk is 21? This app has support KitKat also, but not the framework installation